runtime/shell/src/lib.rs

```diff
@@ -370,3 +376,34 @@ cumulus_pallet_parachain_system::register_validate_block! {
     BlockExecutor = Executive,
     CheckInherents = CheckInherents,
 }
+
+pub struct BaseCallFilter;
+impl Contains<RuntimeCall> for BaseCallFilter {
+    fn contains(call: &RuntimeCall) -> bool {
+        if let RuntimeCall::PolkadotXcm(xcm_method) = call {
+            return match xcm_method {
+                pallet_xcm::Call::execute { .. }
+                | pallet_xcm::Call::teleport_assets { .. }
+                | pallet_xcm::Call::reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_teleport_assets { .. }
+                | pallet_xcm::Call::__Ignore { .. } => false,
+                pallet_xcm::Call::force_xcm_version { .. }
+                | pallet_xcm::Call::force_default_xcm_version { .. }
+                | pallet_xcm::Call::force_subscribe_version_notify { .. }
+                | pallet_xcm::Call::force_unsubscribe_version_notify { .. }
+                | pallet_xcm::Call::send { .. } => true,
```

Comment:
Does it mean anyone can call "send"? Btw, root is not filtered by BaseFilter.

Reply:
Yes, it's ok, if someone wanna `withdraw` asset from sovereign account on relay chain, it will failed because he isn't the root account, can not represent the chain